packages/components/src/ResourcePicker/ResourcePicker.component.js

```diff
@@ -1,25 +1,37 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 import { VirtualizedList } from '../';
 import Resource from './Resource';
+import Toolbar from './Toolbar';
 
-export default function ResourcePicker({ collection }) {
+import theme from './ResourcePicker.scss';
+
+export default function ResourcePicker({ collection, toolbar }) {
 	console.warn(
 		"UNSTABLE WARNING: The 'ResourcePicker' and all the sub components aren't ready to be used in Apps. Code can (will) change outside the release process until it's ready.",
 	);
 
 	return (
-		<div>
-			<VirtualizedList
-				collection={collection}
-				type="resource"
-				rowRenderers={{ resource: Resource }}
-				rowHeight={60}
-			/>
+		<div className={classNames('resource-picker', theme['resource-picker'])}>
+			{toolbar && <Toolbar {...toolbar} />}
```

Comment:
Question : Do we really want to go on this path again ?
I mean having a component that is controlled only by props, and have more complicated props for that.

What about writing that ?
```javascript
function PreparationPicker(props) {
	return <ResourcePicker.Container>
		<ResourcePricker.Toolbar {...myHardCodedProps} />
		<ResourcePicker.List {...listProps} />
	</ResourcePicker.Container>
}
```

And perhaps a default composed version that would contains default toolbar with everything to lighten the toolbar props.
(non blocking comment, because not sure it's in your PR scope)